testing/opensmtpd-extras/APKBUILD

```diff
@@ -0,0 +1,134 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=opensmtpd-extras
+pkgver=201703132115
+pkgrel=0
+pkgdesc="OpenSMTPD addons"
+url="https://opensmtpd.org/"
+arch="all"
+license="ISC"
+depends="libevent"
+depends_filter_monkey="libressl"
```

Comment:
All custom global variables (i.e. variables not specified by abuild) should be prefixed with an underscore (i.e. `_depends_filter_monkey`).